app/presenters/hyrax/admin_set_presenter.rb

```diff
@@ -3,5 +3,22 @@ class AdminSetPresenter < CollectionPresenter
     def total_items
       ActiveFedora::SolrService.count("{!field f=isPartOf_ssim}#{id}")
     end
+
+    # AdminSet cannot be deleted if default set or non-empty
+    def disable_delete?
+      total_items > 0 || default_set?
+    end
+
+    # Message to display if deletion is disabled
+    def disabled_message
+      return I18n.t('hyrax.admin.admin_sets.delete.error_default_set') if default_set?
+      return I18n.t('hyrax.admin.admin_sets.delete.error_not_empty') if total_items > 0
+    end
+
+    private
+
+      def default_set?
```

Comment:
Since this method is identical to the one implemented in the model behavior module above, can the presenter make use of that rather than reproducing it? I'm wondering how we might limit how many spots in the code need to be aware of `AdminSet::DEFAULT_ID`, and how much of that we can concentrate in the behavior.